File: tests/testdata/codegen/simple-range-pp/go/simple-range.go
package main

import (
	"fmt"

	"github.com/pulumi/pulumi-random/sdk/v4/go/random"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		var numbers []*random.RandomInteger
		for index := 0; index < 2; index++ {
			key0 := index
			val0 := index
			__res, err := random.NewRandomInteger(ctx, fmt.Sprintf("numbers-%v", key0), &random.RandomIntegerArgs{
				Min:  pulumi.Int(1),
				Max:  pulumi.Int(val0),
				Seed: fmt.Sprintf("seed%v", val0),
			})
			if err != nil {
				return err
			}
			numbers = append(numbers, __res)
		}
		ctx.Export("first", numbers[0].ID())
		ctx.Export("second", numbers[1].ID())
		return nil
	})
}

File: pkg/codegen/dotnet/testdata/simple-range-pp/simple-range.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using Random = Pulumi.Random;

return await Deployment.RunAsync(() => 
{
    var numbers = new List<Random.RandomInteger>();
    for (var rangeIndex = 0; rangeIndex < 2; rangeIndex++)
    {
        var range = new { Value = rangeIndex };
        numbers.Add(new Random.RandomInteger($"numbers-{range.Value}", new()
        {
            Min = 1,
            Max = range.Value,
            Seed = $"seed{range.Value}",
        }));
    }
    return new Dictionary<string, object?>
    {
        ["first"] = numbers[0].Id,
        ["second"] = numbers[1].Id,
    };
});

